lib/cert/runner.rb

```diff
@@ -81,8 +81,8 @@ def certificates
     # The kind of certificate we're interested in
     def certificate_type
       cert_type = Spaceship.certificate.production
-      cert_type = Spaceship.certificate.development if Cert.config[:development]
       cert_type = Spaceship.certificate.in_house if Spaceship.client.in_house?
+      cert_type = Spaceship.certificate.development if Cert.config[:development] || (Cert.config[:development] && Spaceship.client.in_house?)
```

Comment:
Oh you're right, sorry, we can ditch the second part, actually just moving the development line to last line will do. I'll fix.